roles/deploy_jhub/tasks/main.yml

```diff
@@ -11,6 +11,16 @@
     namespace: kube-system
     chart_ref: "/tmp/cloud-provider-openstack/charts/cinder-csi-plugin"
 
+- name: Install Openstack Controller Manager
```

Comment:
Do we `wget` these into `tmp` before running this?

Reply:
It's in the same repo as the cinder installation just above, so we use that